<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialog</class>
 <widget class="QDialog" name="NewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>463</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>New</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="imageWidth">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="imageWvalue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="value">
    <double>800.000000000000000</double>
   </property>
   <property name="displayIntegerBase" stdset="0">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="imageHeight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="imageHvalue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="value">
    <double>600.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="imageRes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Resolution:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="imageRvalue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imagePreset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Preset Size:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="imagePresetCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>(none)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100 x 100 pixels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>640 x 480 pixels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>800 x 600 pixels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024 x 768 pixels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1366 x 768 pixels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1600 x 900 pixels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1920 x 1080 pixels</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="imageWHcombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>150</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Pixels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inches</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="imageResCombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>150</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Pixels/cm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pixels/inch</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Size in Pixels/Memory</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="pixelWvalue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>69</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>9000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="pixelHvalue">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>69</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>9000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="memoryValue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>69</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <double>9999999.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>331</width>
     <height>311</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="title">
    <string>New Size</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="lockedRatioButton">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>76</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Icons.qrc">
      <normaloff>:/icons/icons/linked.png</normaloff>:/icons/icons/linked.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="BrushTypeComboBox" name="backgroundColorComboBox" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>150</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="ImagePositionWidget" name="positionWidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="positionLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>buttonBox</zorder>
  <zorder>imageWidth</zorder>
  <zorder>imageWvalue</zorder>
  <zorder>imageHeight</zorder>
  <zorder>imageHvalue</zorder>
  <zorder>imageRes</zorder>
  <zorder>imageRvalue</zorder>
  <zorder>imagePreset</zorder>
  <zorder>imagePresetCombo</zorder>
  <zorder>imageWHcombo</zorder>
  <zorder>imageResCombo</zorder>
  <zorder>groupBox</zorder>
  <zorder>backgroundLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrushTypeComboBox</class>
   <extends>QWidget</extends>
   <header>brushtypecombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImagePositionWidget</class>
   <extends>QWidget</extends>
   <header>imagepositionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
